src/iexamine.cpp

```diff
@@ -1537,63 +1547,8 @@ void iexamine::flower_cactus( player &p, const tripoint &examp )
 
     g->m.furn_set( examp, f_null );
 
-    item cactus_pad = item( "cactus_pad" );
-    if( p.can_pickWeight( cactus_pad, true ) &&
-        p.can_pickVolume( cactus_pad, true ) ) {
-        p.i_add( cactus_pad );
-        p.add_msg_if_player( _( "You harvest: %s." ), cactus_pad.tname() );
-    } else {
-        g->m.add_item_or_charges( p.pos(), cactus_pad );
-        p.add_msg_if_player( _( "You harvest and drop: %s." ), cactus_pad.tname() );
-    }
-}
-/**
- * It's a flower, drink nectar if your able to.
- */
-void iexamine::flower_bluebell( player &p, const tripoint &examp )
-{
-    if( dead_plant( true, p, examp ) ) {
-        return;
-    }
-
-    drink_nectar( p );
```

Comment:
Why did you remove the ability to drink nectar for bluebells, tulips and spurge?

Reply:
I didn't remove that, it's still working, just done via another examine function.

To be honest the whole concept of a human-sized creature getting meaningful amounts of nutrition from drinking normal flower nectar (and especially being able to get unlimited amount of nectar from one flower) seems very unrealistic to me, but that's worth a separate discussion.